app/models/spotlight/analytics/ga.rb

```diff
@@ -37,8 +37,8 @@ def dimension_filter(path)
           filter: Google::Analytics::Data::V1beta::Filter.new(
             field_name: 'pagePath',
             string_filter: Google::Analytics::Data::V1beta::Filter::StringFilter.new(
-              match_type: :BEGINS_WITH,
-              value: path.to_s
+              match_type: :PARTIAL_REGEXP,
```

Comment:
FULL_REGEXP works for data and martin-wong as well. Went back and forth about this and decided partial would probably be safer.